### DISCUSSION TITLE
MapOs na versão antiga?

### DISCUSSION THREAD

**Discussion by Felipebzzz** [RamonSilva20/mapos#2326] [General]
Boa tarde a todos. Alguem teria uma versão anterior a do MapOs? Não consigo achar para download e tem arquivos que necessitava, e que só estão presente na versão antiga

**Comment by pwrslv3** [comment]
https://github.com/RamonSilva20/mapos/releases

Só procurar a versão e clicar em "Assets" no fim do descritivo que ele abre a opção pra fazer o download
